### ISSUE TITLE
Bug: 1/2 of Double Fed rounds cannot be reloaded back into the magazine

### ISSUE THREAD

**Issue by YimmestOfYams** [SmidgeonE/Stovepipe#11]
I appear to have found a bug that results in one of the two rounds from a double feed glitching out. Doing a new round of repositioning rounds and casings, but this happens even without any user-side changes.

To replicate, simply download the latest version of stovepipe, spawn any assault/battle rifle or carbine or your choice, shoot and obtain a double feed. After shaking the rounds free or having them drop free of their own volition, the affected round can't be picked up ordinarily, only with the point and grab or the grabbity gloves, and it cannot be loaded back into the magazine. It will clank and clunk against the magazine or phase right through if easy loading is enabled, like it's suddenly not considered a bullet and is now just a basic prop.

Observed Weapon Categories w/ this Glitch:
Assault Rifles
Battle Rifles
Carbines
Antimaterial Rifles (Inconsistent here. I can't reliably make it happen, but it does.)

Weapon Categories Unaffected:
Semi-auto Pistols

These are just the weapon categories I observed, but there could very well be more.

In the event that this is a conflict, I can get you a list of addons and you can tell me anything especially problematic.

Or if this is a case of a busted config file, I can send that as well. Just let me know what will help more.

**Comment by SmidgeonE** [comment]
yeah this is an issue with double feed itself, will look into this
